javascript/src/main/pages/AnalyzeMessageData/CountMessagesByUser.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import BootstrapTable from 'react-bootstrap-table-next';
+import useSWR from "swr";
+import {useAuth0} from "@auth0/auth0-react";
+import {fetchWithToken} from "../../utils/fetch";
+import aggregateUserMessageCount from "../../utils/aggregateUserMessageCount";
+
+const CountMessagesByUser = () => {
+    const columns = [{
+        dataField: 'name',
+        text: 'user'
+    } , {
+        dataField: "count",
+        text: 'Message Count'
+    }];
+    const { getAccessTokenSilently: getToken } = useAuth0();
+    const { data: slackUsers } = useSWR(["/api/slackUsers", getToken], fetchWithToken);
+    const{ data: messages} = useSWR(["/api/members/messages/allmessages", getToken], fetchWithToken);
```

Comment:
Just trying to keep the formatting consistent with the above two lines :)

```suggestion
    const { data: messages } = useSWR(["/api/members/messages/allmessages", getToken], fetchWithToken);
```